ResearchKit/Common/ORKTask.h

```diff
@@ -167,6 +167,13 @@ ORK_AVAILABLE_DECL
 - (ORKTaskProgress)progressOfCurrentStep:(ORKStep *)step withResult:(ORKTaskResult *)result;
 
 /**
+ Validates the task parameters.
+ 
+ This method should be used to check that all the task parameters are correct. E.g., `ORKOrderedTask` makes sure that all its step identifiers are unique. It should throw an exception if validation fails.
```

Comment:
Let's wrap this text more in line with other files in here.
Also, this is a protocol method, so this documentation needs to describe what an implementer should do, in addition to what the caller should do.

In this case, throw an exception on error, because an invalid task is considered unrecoverable.

That's actually an assumption we should consider changing. A validation method like this should probably return a boolean and an error, rather than throw an exception.